crates/node/core/src/node_config.rs

```diff
@@ -22,6 +25,11 @@ use reth_storage_errors::provider::ProviderResult;
 use std::{net::SocketAddr, path::PathBuf, sync::Arc};
 use tracing::*;
 
+/// The default prometheus recorder handle. We use a global static to ensure that it is only
+/// installed once.
+pub static PROMETHEUS_RECORDER_HANDLE: LazyLock<PrometheusHandle> =
+    LazyLock::new(|| prometheus_exporter::install_recorder().unwrap());
```

Comment:
Not sure why this is here, it should be in `crates/node/metrics/src/recorder.rs`